pkg/cfn/builder/vpc.go

```diff
@@ -71,9 +76,24 @@ func (c *ClusterResourceSet) addSubnets(refRT *gfn.Value, topology api.SubnetTop
 	}
 }
 
+func (c *ClusterResourceSet) addNATGateways() {
+
+	switch c.spec.VPC.NATGateway {
+
+	case "highly-available":
+		c.haNAT()
+	case "single":
+		c.singleNAT()
+	case "disabled":
+		c.noNAT()
+	default:
+		c.singleNAT()
```

Comment:
I think this should be an error, the default should be set in [`pkg/apis/eksctl.io/v1alpha5/defaults.go`](https://github.com/weaveworks/eksctl/blob/master/pkg/apis/eksctl.io/v1alpha5/defaults.go).